apps/src/StudioApp.js

```diff
@@ -538,6 +539,15 @@ StudioApp.prototype.init = function (config) {
   this.emit('afterInit');
 };
 
+/**
+ * Used by tests to do some cleanup once test is finished.
+ */
+StudioApp.prototype.cleanup = function () {
+  if (this.changeListener) {
+    Blockly.removeChangeListener(this.changeListener);
+  }
+};
+
```

Comment:
Not sure if I should name this something more clearly test only. Could instead do something like
```
window.__TestInterface__ = window.__TestInterface__ || {};
window.__TestInterface__.studioAppCleanup = function (studioApp) {
  Blockly.removeChangeListener(studioApp.changeListener);
} 
```
which might be better.